documentation/source/user_section/tutorials/registration-to-template/lumbar-registration/labeling.rst

```diff
@@ -0,0 +1,74 @@
+.. _manual-labeling-of-lumbar-landmarks:
+
+Adding landmark labels for template matching
+############################################
+
+Typically, registering an image to the PAM50 template involves using ``sct_label_vertebrae`` to automatically label vertebrae, then selecting 2 intervertebral disc labels to use for matching with the PAM50 template. However, using discs as registration landmarks presents a dilemma: How do we handle variability in the position of the cauda equinea relative to the L1-L2 disc?
+
+Notably, in the PAM50 template, the conus medullaris (i.e. the terminal end of the spinal cord) is aligned with the L1-L2 disc. However, for your subjects, the spinal cord may end above or below this point. So, if registration were based on disc landmarks alone, then the tapered region of the spinal cord may end up misaligned with the template.
+
+To correct for this, the PAM50 template provides a cauda equinea label (specifically, the conus medullaris) as a registration landmark. By creating a similar label in your subject data, you can align the terminal end of the spinal cord between your subject and the PAM50 template. However, this comes with the necessary tradeoff of potentially misaligning the discs.
+
+Deciding which landmarks to use (discs, conus medullaris)
+=========================================================
+
+Registration can be performed with either one or two labels. So, given the tradeoffs between the aligning the discs and/or the conus medullaris, the question becomes: Should registration be done with 1 disc label, 1 conus medullaris label, or a combination of the two?
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lumbar-registration/label-comparison.png
```

Comment:
Nice figure!

Two minor ideas:
- we could add the `17` and `60` labels
- we could add green and red arrows highlight pros and cons areas

<img width="1273" alt="image" src="https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/39456460/ccb2232a-bf4f-4c92-90a1-4e94258b627d">

Reply:
Ooh, these additions are absolutely gorgeous! Thank you for adding them.